indico/modules/search/controllers.py

```diff
@@ -66,6 +66,13 @@ def _process(self, page, q, type, **params):
         }
 
 
+class RHAPISearchPlaceholders(RH):
+    def _process(self):
+        search_provider = get_search_provider()
+        placeholders = search_provider().get_placeholders()
+        return jsonify(placeholders if not PlaceholderSchema(many=True).validate(placeholders) else [])
```

Comment:
I don't think we really need this validation - it's data coming from the plugin, that's most likely fully static - so unlike data returned from an actual search engine with extra metadata we don't want to show, I'd expect plugins to always return clean data here.